src/data/loader.js

```diff
@@ -2,7 +2,7 @@ const importer = require.context('./', true, /\.(yml|yaml)/);
 const keys = [...new Set([...importer.keys()])];
 
 module.exports = keys.reduce((prev, cur) => {
-  const fileName = cur.replace(/\.(yaml|yml)/, '');
+  const fileName = cur.replace(/\.(yaml|yml)/, '').replace('./', '');
```

Comment:
Why is there 2 calls to `.replace`?

Reply:
I want to remove ./ in front of the filename.